packages/server/graphql/private/mutations/setIsFreeMeetingTemplate.ts

```diff
@@ -8,10 +8,10 @@ const setIsFreeMeetingTemplate: MutationResolvers['setIsFreeMeetingTemplate'] =
   const r = await getRethink()
   // VALIDATION
   if (!templateIds?.length && !templateNames?.length) {
-    return {error: {message: 'Must provide template ids or names'}}
+    throw new Error('Must provide template ids or names')
```

Comment:
-1 This will result in the query failing and returning a GraphQL error, so instead of
```
data: {
  error: {
    title: ...
    message: ...
  }
}
```
we will get
```
data: undefined
error: [some backtrace]
```
which is not what we want.